dnf-behave-tests/dnf/history-list-dnf5.feature

```diff
@@ -0,0 +1,147 @@
+# This tests are somehow redundat for dnf4. the features
+# are tested for history command. Example:
+# history list last-1 has the same behavior as history last-1
+# in dnf4
+#
+# This feature file can be merged with history-list.feature
+# in the future if the behavior of history command will change
+# assuming list by default
+@dnf5
+Feature: history list list
+
+Background:
+Given I use repository "dnf-ci-fedora"
+  # create some history to start with
+  And I successfully execute dnf with args "install abcde basesystem"
+  And I successfully execute dnf with args "remove abcde"
+  And I successfully execute dnf with args "install nodejs"
+
+
+Scenario: history list last-1
+ When I execute dnf with args "history list last-1"
+ Then the exit code is 0
+  And stdout is history list
+      | Id | Command | Action  | Altered |
+      | 2  |         | Removed | 3       |
```

Comment:
Case in point, this test is already in the `history-list.feature` file. I wouldn't like to go over the two files and try to match which test is there, what's missing and whats actually redundant...